internal/services/v1/experimental_test.go

```diff
@@ -1426,3 +1426,204 @@ func TestExperimentalDependentRelations(t *testing.T) {
 		})
 	}
 }
+
+func TestExperimentalComputablePermissions(t *testing.T) {
+	tcs := []struct {
+		name             string
+		schema           string
+		definitionName   string
+		relationName     string
+		filter           string
+		expectedCode     codes.Code
+		expectedError    string
+		expectedResponse []*v1.ExpRelationReference
+	}{
+		{
+			name:           "invalid definition",
+			schema:         `definition user {}`,
+			definitionName: "invalid",
+			expectedCode:   codes.FailedPrecondition,
+			expectedError:  "object definition `invalid` not found",
+		},
+		{
+			name:           "invalid relation",
+			schema:         `definition user {}`,
+			definitionName: "user",
+			relationName:   "invalid",
+			expectedCode:   codes.FailedPrecondition,
+			expectedError:  "relation/permission `invalid` not found",
+		},
+		{
+			name: "basic",
```

Comment:
While I do understand this expected response from the perspective of the reachability graph (`user:...`), it seems inconsistent for an API called `ExperimentalComputablePermissions`. I wouldn't expect this to return relations, and so it also wouldn't make sense the API has a field `IsPermission`.

It's not the same with `ExperimentalDependentRelations` because the name of the RPC explicitly states "Relations" so it fits both relations and permissions.

The options that come to mind are:
- create a type `ExPermissionReference` that drops that field, and filter for permissions in the controller
- rename the RPC to something like `ExperimentalComputableRelations` as it could return both.

Reply:
I left it named permissions because that's how people will primarily use it; we could rename it, but I felt it was more discoverable that way. Both functions return both.

Reply:
It could confuse folks, but it's not a big deal anyway, just my 2 cents